sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveContext.scala

```diff
@@ -57,8 +57,10 @@ class LocalHiveContext(sc: SparkContext) extends HiveContext(sc) {
 
   /** Sets up the system initially or after a RESET command */
   protected def configure() {
-    setConf("javax.jdo.option.ConnectionURL",
-      s"jdbc:derby:;databaseName=$metastorePath;create=true")
+    if (getConf("javax.jdo.option.ConnectionURL", "").equals("")) {
```

Comment:
after rebase  this code block has been removed,  the purpose is to avoid hard code.